Feature/merge activity composer and highlight

slickexo2 opened the pull request.

Commit:
add activity-composer

rdenarie commented on packaging/src/main/assemblies/assembly-addon.xml:
```diff
@@ -0,0 +1,59 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+    <id>activity-composer-configuration</id>
```
```suggestion
    <id>functional-configuration</id>
```

rdenarie commented on pom.xml:
```diff
@@ -0,0 +1,87 @@
+    <parent>
+        <artifactId>addons-parent-pom</artifactId>
+        <groupId>org.exoplatform.addons</groupId>
+        <version>8</version>
```
```suggestion
        <version>9</version>
```

rdenarie commented on pom.xml:
```diff
@@ -0,0 +1,87 @@
+                <type>war</type>
+            </dependency>
+			
+			<dependency>
```
```suggestion
	<dependency>
```

rdenarie commented on portlets/src/main/webapp/WEB-INF/web.xml:
```diff
@@ -0,0 +1,29 @@
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd">
+
+    <display-name>activity-composer-configuration-portlets</display-name>
```
```suggestion
    <display-name>functional-configuration-portlets</display-name>
```

Commit:
start adding highlight


no highlights


highligth


Update portlets/src/main/webapp/WEB-INF/web.xml

Co-Authored-By: rdenarie <romain.denarie@exoplatform.com>
Update pom.xml

Co-Authored-By: rdenarie <romain.denarie@exoplatform.com>
Update pom.xml

Co-Authored-By: rdenarie <romain.denarie@exoplatform.com>
Update packaging/src/main/assemblies/assembly-addon.xml

Co-Authored-By: rdenarie <romain.denarie@exoplatform.com>
